deploy/release.groovy

```diff
@@ -56,14 +56,6 @@ def cd (b){
         sh 'npm run build'
     }
 
-    stage('unit test'){
```

Comment:
MMh..I would let it there. It is not taking that much time and testing it again is not a bad thing :)